config/management/src/validator_config.rs

```diff
@@ -42,10 +42,13 @@ impl ValidatorConfig {
         fullnode_address: NetworkAddress,
         validator_address: NetworkAddress,
         reconfigure: bool,
+        disable_address_validation: bool,
     ) -> Result<Transaction, Error> {
-        // Verify addresses
-        validate_address("validator address", &validator_address)?;
-        validate_address("fullnode address", &fullnode_address)?;
+        if !disable_address_validation {
+            // Verify addresses
+            validate_address("validator address", &validator_address)?;
+            validate_address("fullnode address", &fullnode_address)?;
```

Comment:
Okay, so now that I've looked at this and I'm thinking about it.  We should probably not validate addresses when we rotate the keys.  The addresses were already put in previously, and rotating a key is not at the fault of the current operator.  Only when an operator puts in the non-key information, should it validate that it's correct.  Thoughts?

Reply:
Yeah, I basically did this change with the mindset of making as few code changes as possible. Given that they all eventually call into the set-validator-config command, I figured we could just propagate the flag.  I was trying to avoid refactoring everything. However, it might make sense to consider this now. Let's hold off until the original has landed.

Reply:
actually, I would argue it is nice to have this validation... maybe it is over kill, I dunno, but if we have a flag to disable it... I think it is good.